src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Frame.cs

```diff
@@ -658,6 +679,17 @@ private void VerifyAndUpdateWrite(int count)
             _responseBytesWritten += count;
         }
 
+        private bool IsLastWrite()
```

Comment:
Instead of `IsLastWrite` change to something like `CheckLastWrite`. It would include `_abortedCts = null;` if necessary.